logger.go

```diff
@@ -9,3 +9,10 @@ var Logger = golog.Global()
 
 // Debug is helpful to turn on when the library isn't working quite right.
 var Debug = false
+
+type ILogger interface {
+	Debug(...interface{})
```

Comment:
We only needed `Warn` and `Fatal`, but it felt less surprising to me to at least add debug + info.